scripts/make-helpers.sh

```
@@ -17,27 +19,133 @@ hint() {
-  "${cmd[@]}"
-  local rc=$?
+  # Avoid failing captures on shells without process substitution
+  if (true) 2>/dev/null; then
+    "${cmd[@]}" > >(tee "$out_file") 2> >(tee "$err_file" >&2)
+  else
+    "${cmd[@]}" 1>"$out_file" 2>"$err_file"
+  fi
+  rc=$?
   set -e
+
   local t1=$(date +%s)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)